kbase-extension/static/kbase/css/kbaseNarrative.css

```diff
@@ -241,6 +241,57 @@
     padding: 8px;
 }
 
+.kb-staging-table-import {
+    position: relative;
+}
+
+.kb-staging-table-import__button {
+    width: 160px;
+    height: 42px;
+    border-radius: 4px;
+    font-family: Public Sans, Arial, sans-serif;
```

Comment:
is this from figma? I don't think I've even heard of Public Sans! 😱 

(Ah, it's a google font)